packages/canary-react/src/component-tests/IcTreeView/IcTreeView.cy.tsx

```diff
@@ -0,0 +1,276 @@
+    cy.compareSnapshot({
+      name: "disabled-tree-item",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.006),
+    });
+  });
+
+  it("should render with selected tree item", () => {
+    mount(BasicTreeView({}, { selected: true }));
+
+    cy.checkHydrated(TREE_VIEW);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "selected-tree-item",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.01),
+    });
+  });
+
+  it("should render with tree item as link", () => {
+    mount(BasicTreeView({}, { href: "#" }));
+
+    cy.checkHydrated(TREE_VIEW);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "link-tree-item",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.01),
+    });
+  });
+
+  it("should render with expanded tree item", () => {
+    mount(<BasicTreeView />);
+
+    cy.findShadowEl(TREE_ITEM, TREE_ITEM_CONTENT).eq(1).click();
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "expanded-tree-item",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.016),
+    });
+  });
+
+  it("should render with truncated text", () => {
+    mount(
+      <div
+        style={{
+          width: "200px",
+          padding: "16px",
+        }}
+      >
+        <IcTreeView heading="Limited edition menu with extras">
+          <IcTreeItem label="Coffee with lots of extra milk" />
+          <IcTreeItem label="Tea">
+            <IcTreeItem label="Earl Grey with milk on the side" />
+            <IcTreeItem label="Chai" />
+          </IcTreeItem>
+          <IcTreeItem label="Hot Chocolate" />
+        </IcTreeView>
+      </div>
+    );
+
+    const TEXT_OVERFLOW = "text-overflow";
+    const ELLIPSIS = "ellipsis";
+
+    cy.findShadowEl(TREE_ITEM, TREE_ITEM_CONTENT).eq(1).click();
+
+    cy.findShadowEl("ic-tree-view", ".tree-view-header").should(
+      HAVE_CSS,
+      TEXT_OVERFLOW,
+      ELLIPSIS
+    );
+
+    cy.findShadowEl(TREE_ITEM, ".tree-item-label")
+      .eq(0)
+      .should(HAVE_CSS, TEXT_OVERFLOW, ELLIPSIS);
+
+    cy.findShadowEl(TREE_ITEM, ".tree-item-label")
+      .eq(2)
+      .should(HAVE_CSS, TEXT_OVERFLOW, ELLIPSIS);
+
+    cy.checkA11yWithWait();
+    cy.compareSnapshot({
+      name: "truncated-text",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.032),
+    });
+  });
+});
+
+describe("IcTreeView visual regression tests in high contrast mode", () => {
+  before(() => {
+    cy.enableForcedColors();
+  });
+
+  after(() => {
+    cy.disableForcedColors();
+  });
+
+  afterEach(() => {
+    cy.task("generateReport");
+  });
+
+  it("should render", () => {
+    mount(<BasicTreeView />);
+
+    cy.checkHydrated(TREE_VIEW);
+
+    cy.compareSnapshot({
+      name: "default-high-contrast",
+      testThreshold: setThresholdBasedOnEnv(DEFAULT_TEST_THRESHOLD + 0.015),
+    });
+  });
+
+  it("should render with disabled tree item", () => {
```

Comment:
The disabled tree view item doesn't render different to non-disabled tree view items in high contrast. Also noticed the icons don't render, and the focus and selected states aren't visible in high contrast mode.